README.md

```diff
@@ -67,6 +67,11 @@ helm upgrade --install aws-node-termination-handler \
   eks/aws-node-termination-handler
 ```
 
+Basic installation without Helm:
+```sh
+kubectl apply -f https://github.com/aws/aws-node-termination-handler/releases/download/v1.2.0/aws-node-termination-handler-v1.2.0.yaml 
```

Comment:
small suggestion... I realize why you did it this way (because of how you're generating the files), but having the aws-node-termination-handler name and version in the filename in addition to the releases download URL seems a bit verbose.

In other words, instead of this:

```
kubectl apply -f https://github.com/aws/aws-node-termination-handler/releases/download/v1.2.0/aws-node-termination-handler-v1.2.0.yaml 
```

I think it would be nicer to do this:

```
kubectl apply -f https://github.com/aws/aws-node-termination-handler/releases/download/v1.2.0/all.yaml

Reply:
agreed, I moved the version identifier from the filenames to a directory in the build dir. This should prevent confusion between generation and upload scripts while keeping the asset URLs clean.